src/App.js

```diff
@@ -0,0 +1,9 @@
+import MyComponent from './components/Calculator';
+
+function App() {
+  return (
+    <MyComponent />
+  );
+}
```

Comment:
- [ ] Kindly make your App component class-based and not function-based. This way you will have met all the project requirements.

![image](https://user-images.githubusercontent.com/86318284/190118924-b026af96-1e52-44c7-a700-f5146eecec9c.png)